core/templates/domain/user/user-info.model.ts

```diff
@@ -34,9 +34,8 @@ export class UserInfo {
   _isTopicManager: boolean;
   _isSuperAdmin: boolean;
   _canCreateCollections: boolean;
-  // The default value of the following three properties of 'UserInfo' object
-  // should be null, these default values are used when the user is
-  // not logged in. Check function 'createDefault' for reference.
+  // The following three properties are set to null for 'UserInfo' object when
+  // the user is not logged in. Check function 'createDefault' for reference.
```

Comment:
You don't need the reference to the function (second sentence). See my previous comment about the impl not being relevant. (You only need that part in a comment to the reviewer on the Files Changed tab, and in such comments it might make sense to provide a direct link to the function in question.)

Reply:
oh alright! I'll keep note of this.